test/extended/node_tuning/node_tuning.go

```diff
@@ -111,6 +116,57 @@ var _ = g.Describe("[sig-node-tuning] NTO should", func() {
 
 		err = fmt.Errorf("case: %v\nexpected error got because of %v", g.CurrentSpecReport().FullText(), fmt.Sprintf("stalld service restarted : %v", errWait))
 		o.Expect(err).NotTo(o.HaveOccurred())
+	})
+
+	// OCPBUGS-18052
+	g.It("SNO installation does not finish due to wait for non-existing machine-config [Early]", func() {
+		isSNO, err := isSNOCluster(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if !isSNO {
+			g.Skip("only test on SNO cluster, skipping it ...")
+		}
+
+		var (
+			mcpConfigDaemonset *corev1.Pod
+		)
 
+		ctx := context.TODO()
+		nodeClient := oc.KubeClient().CoreV1().Nodes()
+		firstMasterNode, err := getFirstMasterNode(ctx, nodeClient)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		e2e.Logf("ensure that the status of mcp is on updated state")
+		config, err := e2e.LoadConfig()
+		e2e.ExpectNoError(err)
+		dynamicClient := dynamic.NewForConfigOrDie(config)
+		mcps := dynamicClient.Resource(schema.GroupVersionResource{
+			Group:    "machineconfiguration.openshift.io",
+			Version:  "v1",
+			Resource: "machineconfigpools",
+		})
+		pools, err := mcps.List(context.Background(), metav1.ListOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+		for _, p := range pools.Items {
+			err = waitForUpdatedMCP(mcps, p.GetName())
+			o.Expect(err).NotTo(o.HaveOccurred())
+		}
+
+		e2e.Logf("ensure that the status of co machine-config is availabe state")
```

Comment:
typo: available

Reply:
Hi Mrunal, Thank you for taking time to review my PR, I have fixed the typo error